src/app/containers/settings/connections/connections.controller.js

```diff
@@ -4,18 +4,20 @@ angular.module('transmartBaseUi')
     .controller('ConnectionsCtrl', ['$scope', '$location', 'EndpointService', 'StudyListService', 'AlertService',
         function ($scope, $location, EndpointService, StudyListService, AlertService) {
```

Comment:
e.g. here $scope is not used, right? Or do you think we should still keep it in case there needs to be some watchers added later